xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -770,16 +770,16 @@ int CEdl::RemoveCutTime(int iSeek) const
   return iSeek - iCutTime;
 }
 
-int CEdl::RestoreCutTime(int iClock) const
+double CEdl::RestoreCutTime(double iClock) const
```

Comment:
If you change the data type, then please also change the variable's name. Prefix "i" stands for "int". Applies also to other places of your commit.